src/dackar/similarity/simUtils.py

```diff
@@ -603,3 +608,23 @@ def convertSentsToSynsets(sentList, info=None):
     sentSynsets.append(bestSyn)
   return sentSynsets
 
+def combineListsRemoveDuplicates(list1, list2):
+  """combine two lists and remove duplicates
+
+  Args:
+      list1 (list): the first list of words
+      list2 (list): the second list of words
+
+  Returns:
+      list: the combined list of words
+  """
+  combinedList = list1 + list2
+  seen = set()
+  result = []
+
+  for item in combinedList:
```

Comment:
Do we need a for loop? we can combine lists as: list(set(combined_list))

Reply:
yes, we need to remove the duplicates, but also we need to keep the order of words in the list.